### PR TITLE
Examples on effect, palette randomization for wled

### PR DESCRIPTION
**Pull request by rytilahti**
## Proposed change

Add examples on how to activate random effect & random palette.

Obsoletes #15972

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
Thanks, @rytilahti 👍 

Happy Holidays! 🎄

File: source/_integrations/wled.markdown [hunk 1]
```diff
@@ -122,3 +122,39 @@ This service allows for loading a preset saved on the WLED device.
...
+### Activating Random Effect
+
+You can automate changing the effect using a service call like this:
+
+```yaml
+service: wled.effect
+data:
+  entity_id: light.wled
+  effect: "{{ state_attr('light.wled', 'effect_list') | random }}"
+```
```
Comment (frenck) [inline source/_integrations/wled.markdown:139]:
````suggestion
{% raw %}

```yaml
service: wled.effect
data:
  entity_id: light.wled
  effect: "{{ state_attr('light.wled', 'effect_list') | random }}"
```

{% endraw %}
````

File: source/_integrations/wled.markdown [hunk 2]
```diff
@@ -122,3 +122,39 @@ This service allows for loading a preset saved on the WLED device.
...
+
+In this case (using WLED v0.11.0) there are 54 palettes, so the following service call will activate a random palette by its ID between 0 and 53:
+
+```yaml
+service: wled.effect
+data:
+  entity_id: light.wled
+  palette: "{{ range(0,53) | random }}"
+```
```
Comment (frenck) [inline source/_integrations/wled.markdown:166]:
````suggestion
{% raw %}

```yaml
service: wled.effect
data:
  entity_id: light.wled
  palette: "{{ range(0,53) | random }}"
```

{% endraw %}
````

**Review by frenck — APPROVED** [review]

**Comment by rytilahti** [comment]
Thanks for the suggestions & quick turnaround times, and happy holidays to you too, @frenck!

**Pull request merged**